.github/workflows/checkov.yml

```
@@ -0,0 +1,27 @@
+jobs:
+  checkov-scan:
+    name: Checkov Scan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Run Checkov (All Frameworks)
+        uses: bridgecrew/checkov-action@v3
+        with:
+          directory: ./   
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)